cobalt/android/BUILD.gn

```diff
@@ -44,7 +45,8 @@ android_library("cobalt_apk_java") {
     "//components/embedder_support/android:view_java",
     "//content/public/android:content_java",
     "//content/shell/android:content_shell_apk_java",
-    "//content/shell/android:content_shell_apk_resources",
+
+    #"//content/shell/android:content_shell_apk_resources",
```

Comment:
Extract this out down below to line 59:

```
if(!is_cobalt) {
  deps += [
     "//content/shell/android:content_shell_apk_resources"
  ]
}
```

Reply:
Actually nevermind - this only gets compiled _for_ cobalt configs, correct ? So it can just be removed